parkhangs-webapp/src/features/modal/ModalEventDetail.css

```diff
@@ -130,12 +129,58 @@
     width: 90%;
     display: flex;
 }
+.ModalEventDetail-actionButton{
+    padding: 10px;
+    background-color: #f9f9f9;
+    border: 1px solid #aab9d4;
+    border-radius: 3px;
+    color: #000;
+}
+.react-add-to-calendar__button--light{
+
+}
+.dropdown{
+    position: absolute;
+    margin-top: 0.65rem;
+    margin-left: -17px;
+    padding-top: 1rem;
+    padding-right: 0.8rem;
+    padding: 0.5rem;
+    background-color: #f9f9f9;
+    border: 1px solid #aab9d4;
+    border-radius: 3px;
+    color: #000; 
+
+
+}
+.dropdown > ul {
+    display: inline-block;
+    
+    
+}
+.dropdown > ul > li {
+    position: relative;
+    padding: 0.5rem;
+    margin-bottom: 0.5rem;
+    border: 2px solid #aab9d4;
+    border-radius: 3px;
```

Comment:
rem for lines 165, 166

Reply:
Borders can be static value so I don’t need to switch to rem, or that’s my understanding.